backend/substrapp/tasks/utils.py

```diff
@@ -12,7 +12,7 @@
 
 from kubernetes import client, config
 
-CELERYWORKER_IMAGE = os.environ.get('CELERYWORKER_IMAGE')
+CELERYWORKER_IMAGE = os.environ.get('CELERYWORKER_IMAGE', 'substrafoundation/celeryworker:latest')
```

Comment:
It fixes https://github.com/SubstraFoundation/substra-backend/pull/128 because if the env variable `CELERYWORKER_IMAGE` is not set image name is `None`

Reply:
Got it, thanks.